{"name": "test_unsuccessful_login", "status": "broken", "statusDetails": {"message": "NameError: name 'request' is not defined", "trace": "@pytest.fixture(scope=\"function\", autouse=True)\n    def setup_browser():\n>       browser_name = request.config.getoption('browser_name')\nE       NameError: name 'request' is not defined\n\ntests\\conftest.py:32: NameError"}, "start": 1689778573592, "stop": 1689778573592, "uuid": "62107f60-0113-4820-844d-657280fd13ff", "historyId": "64fa320f84ca13fdf7a1ce2c29801c22", "testCaseId": "64fa320f84ca13fdf7a1ce2c29801c22", "fullName": "tests.test_login#test_unsuccessful_login", "labels": [{"name": "tag", "value": "web"}, {"name": "feature", "value": "Login with invalid data"}, {"name": "story", "value": "Checking if an error is displayed when entering a password that is not in the system"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LAPTOP-OFQSKCIR"}, {"name": "thread", "value": "2768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}